src/types/DashboardItem.ts

```diff
@@ -1,11 +1,11 @@
-import { ReactNode } from 'react';
-
-export type DashboardItemLargeType = {
+export type DashboardItemSmall = {
```

Comment:
Consider moving Props into the files in which the components are created, and name them accordingly;

`DashbaordItemSmallProps`

Reply:
In this case, is it not better to have both in the same file? Since one of them depends on the other.

Reply:
I believe it's fine to import the type. You can use a type-only import to ensure the coupling between the files is loose like this;

```typescript
export type DashboardItemLargeProps = import("./ComponentSmall"). DashboardItemSmall & {

}
```

or if you find it cleaner, import the small-type first on a separate line with `type BlaBlaSmall = import(...);`